extensions/adobe/experience/journeyOrchestration/stepEvents/journeyOrchestrationJourneyFields.example.1.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
example seems to be missing the name, description and version fields

Reply:
Sure, will add another example with all the fields. Just wanted to set a partial one to validate this example also works fine.